parm/config/config.resources

```diff
@@ -623,6 +623,14 @@ elif [ ${step} = "vrfy" ]; then
     fi
     export is_exclusive=True
 
+elif [ ${step} = "fit2obs" ]; then
+
+    export wtime_fit2obs="00:20:00"
+    export npe_fit2obs=3
+    export nth_fit2obs=1
+    export npe_node_fit2obs=1
+    if [[ ${machine} == "WCOSS2" ]]; then export npe_node_fit2obs=3 ; fi
```

Comment:
Why does WCOSS require three full nodes vs only 3 CPU on others?

Reply:
WCOSS2 only uses 1 node (npe_fit2obs / npe_node_fit2obs = 3/3 = 1), whereas the R&Ds use 3 nodes (3/1 =3). This is to get enough memory.

Reply:
I always forget it is defined backwards. When we rewrite the resources, we need to improve the variable name.